zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -346,9 +383,14 @@ where
 
                 coinbase_txn: TransactionTemplate::from_coinbase(&coinbase_tx, miner_fee),
 
-                target: empty_string.clone(),
+                target: format!(
+                    "{}",
+                    expected_difficulty
+                        .to_expanded()
+                        .unwrap_or_else(|| ExpandedDifficulty::from(U256::zero()))
```

Comment:
There's this assert that seems concerning: 

> `assert!(self.0 > 0.into(), "Zero difficulty values are invalid")`

https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/work/difficulty.rs#L345

I think we should try returning an error if it's `None` and reevaluate if that causes problems.

Reply:
Yes, the idea here was to return one just as https://github.com/ZcashFoundation/zebra/pull/5659/files/6ba12c388493814f621b28681ad1e223a05001c7#diff-8c6198fdf18f8b2691f9c876d3a10f653827cee5a8cce4e8496451d03886c122 but i forgot to update.

Whatever is the case i will like to solve this 2 issues with the mocked service, however i am not totally sure how to do it yet.

Reply:
It might be the same as [here](https://github.com/ZcashFoundation/zebra/pull/5630/files#diff-39f9960d69871ed9095292c4272532f519193be4bb30b5e56c61c3cc4c706c16) or [here](https://github.com/ZcashFoundation/zebra/pull/5630/files#diff-78bcac22b320634c3859f8282d82427e7fa135c34607223ffdaad3c71c7d8379) but for `ChainInfo` with `ExpandedDifficulty::from(U256::one())` as the response.